js/controllers/BylawsCtrl.js

```diff
@@ -7,14 +7,14 @@ angular.module(ctrl_name, []).controller(ctrl_name, ['$scope', function($scope)
             internal_title: 'About Text',
             body: 'The Bylaws of RPI Ambulance is a supporting document. It covers the duties and ' +

```

Comment:
Please change to "The Bylaws of RPI Ambulance cover the duties..." basically removing "is a supporting document..."

Reply:
Fixed with latest commit.